### ISSUE TITLE
searching is "stuck" on liked images only.

### ISSUE THREAD

**Issue by ShadowWizard11** [zanllp/sd-webui-infinite-image-browsing#665]
So, when I click to view images "walk img2img" I see all the images.  as soon as I click to search, it will only search images I have liked, not all the images.  This has suddenly made searching useless.

**Comment by zanllp** [comment]
Did you add the "like" tag to "Exact Match"? Or did you specify the Search Scope? I cannot replicate this issue.

<img width="527" alt="image" src="https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/13fdab01-9644-4fa1-ba48-c5db195547cf">

**Comment by ShadowWizard11** [comment]
First off, thank you for taking the time to reply.  I did make sure that "Like" was not added anywhere.  However I did manage to solve the issue.  Since you were kind enough to reply to me, I thought I would let you know it is resolved, and how I resolved it.  Both for you, and for anyone that may find this in the future.
The solution I just happened by accidently.  I added another pic to my "like" from another folder in the walk section that I had added, and the problem went away.  Don't know if it was because I was in a a different folder or not, but thought I would mention the complete resolution I had.
